nextcloud/Dockerfile.23.0

```diff
@@ -0,0 +1,83 @@
+FROM hoellen/nginx-php:7.4
```

Comment:
According to https://docs.nextcloud.com/server/23/admin_manual/installation/system_requirements.html PHP 8 is recommended, as also mentioned in https://github.com/hoellen/dockerfiles/issues/59

Reply:
This is totally true, it slipped somehow trough wile commiting this.

Reply:
The recommendation of 8.0 is present since Nextcloud 21. Even, there is no `hoellen/nginx-php:8.0` til now. Maybe this can be solved in a different PR (maybe with providing this image).

Reply:
I've opened #61 about PHP 8.0 (and 8.1).